app/models/katello/concerns/smart_proxy_extensions.rb

```diff
@@ -146,17 +147,21 @@ def pulp3_support?(repository)
       def content_pulp3_support?(content_type)
         content_type = content_type.is_a?(String) ? content_type : content_type.model_class::CONTENT_TYPE
         type = Katello::RepositoryTypeManager.find_repository_type content_type
-        type.pulp3_plugin && SmartProxy.pulp_master!.capabilities('Pulp3').try(:include?, type.pulp3_plugin)
+        type.pulp3_plugin && SmartProxy.pulp_master!.capabilities(PULP3_FEATURE).try(:include?, type.pulp3_plugin)
       end
 
-      def pulp3_host!
-        url = self.setting('Pulp3', 'pulp_url')
+      def pulp3_uri!
+        url = self.setting(PULP3_FEATURE, 'pulp_url')
         fail "Cannot determine pulp3 url, check smart proxy configuration" unless url
-        URI.parse(url).host
+        URI.parse(url)
+      end
+
+      def pulp3_host!
```

Comment:
I kept this in for compatibility but I don't think this is used anywhere now btw.